astroid/bases.py

```diff
@@ -38,8 +38,9 @@
     from astroid.constraint import Constraint
 
 
-PROPERTIES = {"builtins.property", "abc.abstractproperty"}
-if PY310_PLUS:
+PROPERTIES = {"builtins.property", "abc.abstractproperty", "functools.cached_property"}
+# enum.property was added in Python 3.11
+if PY311_PLUS:
     PROPERTIES.add("enum.property")
```

Comment:
Should this be based on pylint's ``py-version`` instead ? Not sure if we have the ability to propagate this option to astroid yet. If not I don't know what exactly the design should be. (this is more a question for @DanielNoord)

Reply:
We don't really have an option to...
Since we haven't had anybody complain about this for 2-3 years after making the mistake I wouldn't bother with adding complicated support for it.